README.md

```diff
@@ -4,13 +4,27 @@ This project is a demonstration of how to create a stateless Virtual JSON Graph
 
 ## Getting Started
 
-```
-clone repo
+Clone the repo: `https://github.com/Netflix/falcor-express-demo.git`
+
+``` sh
+cd falcor-express-demo
 npm install
 npm start
-open your browser and visit http://localhost:9090
 ```
 
+Open your browser and visit: [http://localhost:9090](http://localhost:9090)
+
+## Notes
+
+Changing the `index.html` file will refresh the browser automatically using
+[LiveReload][2]. Changing the server `index.js` file will reload the server
```

Comment:
`restart` could be a more appropriate word here than `reload` as per server context. What do you say?